interface/wx/overlay.h

```diff
@@ -27,6 +27,25 @@ class wxOverlay
        example, when the window content has been changed and repainted.
     */
     void Reset();
+
+    /**
+        Sets or unsets constant opacity of the overlay window.
+
+        If @a alpha is `-1`, use per-pixel alpha blending, otherwise use the
+        given alpha value for all pixels.
+
+        @note The overlay window is opaque by default under wxMSW. You have to
```

Comment:
That's not my understanding of how 'wxOverlay` is supposed to work. It's expected that calling `wxDCOverlay::Clear()` will make the overlay transparent.

Reply:
In fact, it works as you expect. Just the wording is misleading and should say something like this instead:
> Drawing on the overlay window is opaque by default under wxMSW...

because this is how `SetLayeredWindowAttributes` works. i.e. it can only apply a constant alpha on the drawing with no alpha blending at all.

As pictures speak louder than words, this is what I mean with rubberbanding as example:

- Rubberbanding (outline) with `SetLayeredWindowAttributes` + `wxALPHA_OPAQUE` (default):

![overlay_opaque_outline](https://github.com/wxWidgets/wxWidgets/assets/7704771/21b6dbeb-05b5-4f09-9113-5894c7cb868d)

- Rubberbanding (filled) with `SetLayeredWindowAttributes` + `wxALPHA_OPAQUE` :

![overlay_opaque](https://github.com/wxWidgets/wxWidgets/assets/7704771/c2400b2a-3740-4ad3-b9c8-e06b93caae7a)

- Rubberbanding (filled) with `SetLayeredWindowAttributes` + alpha (128): 

![overlay_translucent](https://github.com/wxWidgets/wxWidgets/assets/7704771/8022d6c4-ee9b-4314-8154-f092f6f80c9d)

- Rubberbanding (filled) with `UpdateLayeredWindow` + wxGCDC: 

![overlay_translucent_gc](https://github.com/wxWidgets/wxWidgets/assets/7704771/8b3bd91e-eb40-4bfe-b3dd-689ea1dc7417)